pkg/aws/iam.go

```diff
@@ -190,3 +196,27 @@ func unMarshallPolicyDocument(document string) (*PolicyDocument, error) {
 	}
 	return &policyDocument, nil
 }
+
+// getIAMRole returns the role object struct that has attributes of a given role
+func getIAMRole(ctx context.Context, client IAMClient, roleName string) (output *iam.GetRoleOutput, err error) {
+	input := &iam.GetRoleInput{
+		RoleName: &roleName,
+	}
+
+	output, err = client.GetRole(ctx, input)
+	if err != nil {
+		return output, err
+	}
+	log.Println(output.Role)
```

Comment:
**Question:** Any reason in particular you are logging the entire output? This is going to clutter up the test logs, and we actually don't have a spec defined for how we do logging yet.

Reply:
removed as this was the remanent of my test